src/Microsoft.Health.Dicom.Client/DicomWebClientExtensions.cs

```diff
@@ -114,6 +114,7 @@ public static class DicomWebClientExtensions
             int[] frames = null,
             CancellationToken cancellationToken = default)
         {
+            EnsureArg.IsNotNull(dicomWebClient, nameof(dicomWebClient));
```

Comment:
Are null strings valid here for the various uids?

Reply:
Does not appear so

Reply:
addressed